docs/android-core/participants/events.mdx

```diff
@@ -22,197 +22,283 @@ Here are the supported methods:
 
 Triggers an event when any participant joins the meeting.
 
+<Tabs groupId="android-core-participants">
+  <TabItem value="kotlin" label="Kotlin" default>
+
 ```kotlin
     private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onParticipantJoin(participant: DyteMeetingParticipant) {
-            // your code here to handle new participant
+        override fun onParticipantJoin(participant: DyteRemoteParticipant) {
         }
     }
 ```
 
+  </TabItem>
+
+  <TabItem value="java" label="Java" default>
+
+```java
+    DyteParticipantsEventListener listener = new DyteParticipantEventsListener() {
+        @Override
+        public void onParticipantJoin(DyteRemoteParticipant participant) {
+        }
+    };
+```
+
+  </TabItem>
+</Tabs>
+
 ### Participant left
 
 Triggers an event when any participant leaves the meeting.
 
+<Tabs groupId="android-core-participants">
+  <TabItem value="kotlin" label="Kotlin" default>
+
 ```kotlin
     private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onParticipantLeave(participant: DyteMeetingParticipant) {
-            // your code here to handle participant left from meeting
+        override fun onParticipantLeave(participant: DyteRemoteParticipant) {
         }
     }
 ```
 
+
+  </TabItem>
+
+  <TabItem value="java" label="Java" default>
+
+```java
+    DyteParticipantsEventListener listener = new DyteParticipantEventsListener() {
+        @Override
+        public void onParticipantLeave(DyteRemoteParticipant participant) {
+        }
+    };
+```
+
+
+  </TabItem>
+</Tabs>
+
+
 ### Participants update
 
 Triggers an event whenever there is any change in the `meeting.participants` object. This includes any updates to participant lists or changes in individual participant within those lists.
 
+<Tabs groupId="android-core-participants">
+  <TabItem value="kotlin" label="Kotlin" default>
+
 ```kotlin
     private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onUpdate(participants: DyteParticipants) {
+        override fun onAllParticipantsUpdated(allParticipants: List<DyteParticipant>) {
```

Comment:
```suggestion
        override fun onUpdate(participants: DyteParticipants) {
```

The old `onUpdate` callback was correct. It's a generic callback given for every update in `DyteParticipants` object.